content/feature/Balance Sheet Implementation - Hellhound14.md

```diff
@@ -0,0 +1,40 @@
+|Date|Profit ($)|Loss ($)|Monthly Earnings ($)|Total ($)|

```

Comment:
Look at the headings I used for my issue and your's... They are different.

Reply:
I used them as according to the code I wrote for the balance sheet.

If needed I can change them

I thought it would make the code more relatable to the viewer